go/libkb/device_clone_token.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2018 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package libkb
+
+import "fmt"
+
+const (
+	DefaultCloneTokenValue string = "00000000000000000000000000000000"
+)
+
+type DeviceCloneToken struct {
```

Comment:
`DeviceCloneToken` strikes me as a strange name for something that is not a single token. Maybe something like `DeviceCloneTokens` or `DeviceCloneState` or `CloneState`.